share/doc/homebrew/Tips-N'-Tricks.md

```diff
@@ -136,3 +136,9 @@ inline patches in Vim.
 [homebrew-mode](https://github.com/dunn/homebrew-mode) provides syntax
 highlighting for inline patches as well as a number of helper functions
 for editing formula files.
+
+### Atom
+
+[language-hombrew-formula](https://atom.io/packages/language-hombrew-formula)
```

Comment:
Typo was on the package and has now been fixed https://github.com/josa42/atom-language-homebrew-formula/issues/3#issuecomment-240013203